Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -521,6 +524,36 @@ CastCopy(float * to, const void * from, size_t pixelcount)
   }
 }
 
+// Internal function to convert vectors between RAS and LPS coordinate systems.
+// Dimensions are CXYZT (ITK memory layout)
```

Comment:
If this documentation is relevant to the method/class, should be put in the method's declaration docstring; if it only relevant to those reading/maintaining the code, please move it to the method implementation body (e.g. before any statement).

Reply:
This is a local helper function as the others around it and all the others near the top of the file. It does not have a separate declaration, so I think the comment's location is appropriate. This is the convention that is already used in the short documentation of the other similar functions.